action/action-dsl/src/main/groovy/com/powsybl/action/dsl/ActionDslLoader.groovy

```diff
@@ -236,6 +249,8 @@ class ActionDslLoader extends DslLoader {
 
             shell.evaluate(dslSrc)
 
+            checkIfUndefinedActions(actionsId, rulesDb)
+
```

Comment:
i think rulesDb already contains Rules and Actions, maybe you don't need to save all actions in actionsId.

Reply:
actionsId does not contain Actions in rulesDb but the id of the actions applied by rules (without knowing if they are in rulesDb). checkIfUndefinedActions checks if all the ids actually exist in rulesDb.

Reply:
yes it is possible, but having actionsId allows to test the ids only once if actionsId is a Set